pipeline.yaml

```diff
@@ -67,6 +66,7 @@ Parameters:
     Type: String
     Description: ECS Respository Name (If URI is xxx.dkr.ecr.us-east-1.amazonaws.com/nginx, then nginx)
   ECSRegion:
```

Comment:
I think we can avoid `ECSRegion` from now, as it will be in the same region as CodePipeline.